larray/tests/test_la.py

```diff
@@ -195,16 +195,6 @@ def test_match(self):
         self.assertEqual(sutcode.startswith('A23'), LGroup(['A23', 'A2301']))
         self.assertEqual(sutcode.endswith('01'), LGroup(['A2301', 'A2501']))
 
-    def test_getitem(self):
-        age = Axis('age', '0..10')
-        group = age[':3']
-        self.assertEqual(group.key, slice(None, 3, None))
-        self.assertTrue(group.axis.equals(age))
-
-        group = age[:]
-        self.assertEqual(group.key, slice(None))
-        self.assertIs(group.axis, age)
```

Comment:
this test is slightly different from the one in the "new" test_getitem above, so should be moved there instead of being removed (ie. an explicit test for non-string slices) and/or a test for a slice(None) is nice to have.